games/p.yaml

```
@@ -1483,6 +1483,26 @@
   video:
     youtube: is-9SjYr_WE
 
+- name: PythonPlantsVsZombies
+  originals:
+  - Plants vs. Zombies
+  type: clone
+  repo: 'https://github.com/marblexu/PythonPlantsVsZombies'
+  development: halted
+  status: playable
```

```suggestion
  status: semi-playable
```
it doesn't appear that the game can play through the whole campaign